playbook/app/pb_kits/playbook/pb_table/index.ts

```diff
@@ -2,6 +2,7 @@ import PbEnhancedElement from '../pb_enhanced_element'
 
 const TABLE_WRAPPER_SELECTOR = "[data-pb-table-wrapper]";
 const TABLE_COLLAPSIBLE_WRAPPER_SELECTOR = "[data-pb-table-collapsible-wrapper]";
+const TABLE_COLLAPSIBLE_CELL_SELCTOR = "[data-pb-table-collapsible-cell-id]";
```

Comment:
The selector name is missing an "E" on "_SELCTOR". @nickamantia

Reply:
thank youuu!!!